### PR TITLE
Feat: Alert and Button component

### LINKED ISSUE #16: Feature: Alert component
# Complete Alert component

Feature: Complete Alert component

## Complete alert component with 2 or 3 variants

This Feature will add a new alert component to the docs page.

## Alert tasks (Complete in order)

- [x] Task: Create basic alert component
- [x] Task: Create a filled alert component with all styles eg. warning, success
- [x] Task: create an alert component with only colored outlines.
- [x] Added close button to all alert component
- [x] Add code blocks
- [x] Add usage for alert button in docs

### LINKED ISSUE #17: Feature: Button component
# Complete Button component

Feature: Complete Button component

## Complete Button component with mentioned variants

This Feature will add a new button component to the docs page.

## Button tasks (Complete in order)

- [x] Task: Create basic button component
- [x] Task: Create a button component (style filled colors) with all styles eg. warning, success
- [x] Task: Create a button component (style outlined colors) with all styles eg. warning, success
- [x] Added button component with icon
- [x] Add code snippets for the buttons
- [x] Add documentation page for buttons
- [x] Add button page link to all other pages.

### PR DESCRIPTION
**Pull request by Pratham82**
# Completed Alert and Button component implementation

### 1. Alert component:

- [x] Created a filled alert component with all styles eg. warning, success
- [x] Created an alert component with only colored outlines.
- [x] Created alert component with icons

### 2. Button component:

- [x] Created a filled button component with all styles eg. warning, success
- [x] Created a button component with only colored outlines.
- [x] Created button component with icons
- [x] Create a link button
- [x] Created floating action button component

## How does the solution address the problem

This PR will complete 2nd branch of P0

## Linked issues

Resolves
resolves #16
resolves #17

closes #16 , closes #17

### CODE REVIEW

**Review by Asuraking1n — COMMENTED** [review]
Looks good to me, just suggesting use rem instead of px.

File: src/components/alert/alert.html [hunk 1]
```diff
@@ -9,6 +9,12 @@
     <link rel="stylesheet" href="../../css/main.css" />
     <link rel="stylesheet" href="../../css/docs.css" />
     <link rel="stylesheet" href="./alert.css" />
```
Comment (krgarima) [inline src/components/alert/alert.html]: You can put all of them in a single css file and import that css file.

File: src/components/button/button.html [hunk 2]
```diff
@@ -8,7 +8,13 @@
     <link rel="stylesheet" href="../../css/header.css" />
     <link rel="stylesheet" href="../../css/main.css" />
     <link rel="stylesheet" href="../../css/docs.css" />
-    <link rel="stylesheet" href="./avatar.css" />
+    <link rel="stylesheet" href="./button.css" />
+    <link
+      rel="stylesheet"
+      href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css"
```
Comment (krgarima) [inline src/components/button/button.html]: I don't know the reason but someone suggested in Review session to import the material icon link through CSS.
Syntax: @import 'file'

**Review by krgarima — COMMENTED** [review]
Site looks neat and clean. Just some little suggestions.

File: src/components/alert/alert.html (hunk 1 above)
Comment (Pratham82) [reply src/components/alert/alert.html]: Thanks, I've added common CSS.

File: src/components/button/button.html (hunk 2 above)
Comment (Pratham82) [reply src/components/button/button.html]: Fixed, added the icons url to root.css

**Comment by Pratham82** [comment]
> Site looks neat and clean. Just some little suggestions.

Thanks for the suggestions, I've made the changes

**Comment by Pratham82** [comment]
> Looks good to me, just suggesting use rem instead of px.

Thanks, I'll add it.

**Pull request merged**